### ISSUE TITLE
Feature Request: Be able to describe characters (NPCs) independent from object list

### ISSUE THREAD

**Issue by ainslec** [ainslec/adventuron-issue-tracker#16]
Currently characters are described as a scenery type object. Adventuron should be able to list characters in the object list, or characters in an independent character list part of the screen.

**Comment by hnejfnvjojgrogjwm** [comment]
Suggestions.

I think the string for characters' list should be something like:
{name1} and {name2} are here.
Sounds more natural, "literary", than the formula used for objects.

Additionally I would appreciate it if I can choose not to show the charters' list on the location, as I have specific strings for characters and locations, like in: living room, afternoon: "Jim is eating his lunch and reading a book" that right now is displayed together with the name "Jim" in the list of objects.

**Comment by ainslec** [comment]
Yes, I agree. The way that characters are listed should be configurable.

That would be covered via the existing conspicuous="false" attribute (which would also be on characters).
